src/sharing/Table.tsx

```diff
@@ -110,10 +111,11 @@ class Table extends React.Component<TablePropsWithStyles> {
 
     render() {
         const { allowPublicAccess, allowExternalAccess } = this.props.meta.meta;
```

Comment:
Now that they are optional, we may define default values. `publicAccess=true` and `externalAccess=false`?

Reply:
Sure thing